quinn-proto/src/connection.rs

```diff
@@ -2798,7 +2812,13 @@ pub enum Io {
         packet: Box<[u8]>,
     },
     /// Stop or (re)start a timer
-    TimerUpdate { timer: Timer, update: TimerUpdate },
+    TimerUpdate {
```

Comment:
Unrelated style change.

Reply:
rustfmt does this directly in response to the added enum variant, for some reason.